_webpack/include.js

```diff
@@ -0,0 +1,10 @@
+// Add source map support for proper stack traces
+import 'source-map-support/register';
+
+// Catch all unhandled exceptions and print their stack trace.
+// Required if the handler function is async, as promises
+// can swallow error messages.
+process.on('unhandledRejection', e => {
```

Comment:
I would print the entire `stringified` error object, seeing as it was an unexpected error, the `error.message` would help out in debugging too.

Reply:
AFAICT e.stack will include both stack and message:

```js
> let terribleError = new Error('oh noes!');
> terribleError.toString()
'Error: oh noes!'
> terribleError.message
'oh noes!'
> console.log(terribleError.stack)
Error: oh noes!
    at repl:1:21
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:240:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:468:10)
    at emitOne (events.js:121:20)
    at REPLServer.emit (events.js:211:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
```